### ISSUE TITLE
release v1.1.0-alpha1 uses v1.0.0 importer pod

### ISSUE THREAD

**Issue by davidvossel** [kubevirt/containerized-data-importer#252]
importer pod v1.0.0 is hard coded as the default unless an ENV VAR is set to specify a different one. Our latest CDI release does not set that ENV VAR which means we have a v1.1.0-alpha.1 version using the old v1.0.0 importer pod. 

example manifest.  ENV VARs need to be set in this manifest reflecting the importer pod version the controller should use. 
```
apiVersion: apps/v1beta2
kind: Deployment
metadata:
  name: cdi-deployment
  namespace: default  # update accordingly based on the namespace you are deploying in
  labels:
    cdi.kubevirt.io: ""
    app: containerized-data-importer
spec:
  selector:
    matchLabels:
      app: containerized-data-importer
  replicas: 1
  template:
    metadata:
      labels:
        app: containerized-data-importer
    spec:
      serviceAccountName: cdi-sa
      containers:
      - name: cdi-controller
        image: kubevirt/cdi-controller:v1.1.0-alpha.1
        imagePullPolicy: IfNotPresent
        args: ["-v=1"] # default verbosity; change to 2 or 3 for more detailed logging
---
```

**Comment by jeffvance** [comment]
@davidvossel  I think resolving these issues (#55, #223) using templates will solve this and similar problems. Thoughts?

**Comment by copejon** [comment]
#253 should not be merged, but describes the cause of the problem here. I _think_ that by rebasing master onto v1.1.0-alpha.1 tag, then pushing to master, this will fix the issue.  The cause of this I'm still not sure of.

**Comment by copejon** [comment]
I've downloaded and deployed the v1.1.0-alpha.1 release in minikube.  What I see is that it does use the cdi-importer:v1.1.0-alpha.1 image for the importer pod.

Now, currently, master is in a unstable state because of what I describe in #253.  If you're building from master, then the behavior you're describing is definitely happening.

**Comment by davidvossel** [comment]
> I think resolving these issues (#55, #223) using templates will solve this and similar problems. Thoughts?

if the importer container is a templated value then it will.

**Comment by jeffvance** [comment]
@davidvossel I believe merged pr #246 (and a couple of related minor follow-up prs) address this issue. Let me know if this is true and I'll close it.

**Comment by aglitke** [comment]
This release has been superseded by newer releases which don't exhibit this problem.  wontfix.
